app-creator/src/redux/reducer.ts

```diff
@@ -106,3 +146,12 @@ export const reducer: Reducer<AppCreatorStore, AppCreatorAction | AnyAction> = (
       return state;
   }
 };
+
+/**
+ * Updates DOM element with attributes.
+ */
+function updateUI(widget: HTMLElement, template: { [key: string]: any }) {
+  Object.keys(template[widget.id].uniqueAttributes).forEach((attr: string) => {
```

Comment:
It is discouraged to use `forEach`, so let's replace it with `for (const x of someArr) {...}`. See [go/tsstyle#iterating-arrays](go/tsstyle#iterating-arrays).